sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -1035,7 +1035,7 @@ static size_t _iphc_ipv6_encode(gnrc_pktsnip_t *pkt,
 
         /* copy remaining byteos of flow label */
```

Comment:
```suggestion
        /* copy remaining bytes of flow label */
```

while you are at it?